addon/components/editor-plugins/template-variable-card.js

```diff
@@ -149,30 +154,34 @@ export default class EditorPluginsTemplateVariableCardComponent extends Componen
           const codelistSourceTriple = fullDatastore
             .match(`>${mappingUri}`, 'dct:source', null)
             .asQuads()
+            .next();
+          let codelistSource;
```

Comment:
any reason this is duplicated? looks to me like we can move the whole source fetching logic out of the if statement

Reply:
You cannot move it out of the if statement because it needs to happen only if the variable is a codelist or a location, but I extracted it into a separate function